moodler/config.py

```diff
@@ -0,0 +1,69 @@
+# Ivy
```

Comment:
We should move this configuration into a .env file I think
https://github.com/theskumar/python-dotenv